src/config.js

```diff
@@ -7,257 +7,257 @@ export const intro = {
     ],  
 }
 
-export const questions = {
-    food: {
-        title: 'Eten',
-        questions: {
-            1: {
-                question: 'Laat je graag eten thuis bezorgen?',
-                imgAnswers: false,
-                answers: [
-                    {
-                        answer: 'Zeker, het liefst zou ik elke dag bestellen!',
-                        value: 0
-                    },
-                    {
-                        answer: 'Ja, maar dan alleen een gezonde poké bowl of salade.',
-                        value: 0
-                    },
-                    {
-                        answer: 'Nee, eigenlijk niet.',
-                        value: 0
-                    },
-                ],                    
-            },
-            2: {
-                question: 'Wat zie je tijdens een feestje het liefst op tafel staan?',
-                imgAnswers: true,
-                answers: [
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q2A1.png',
-                            alt: 'Een plank met kaas, brood en druiven',
-                          }
-                    },
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q2A2.png',
-                            alt: 'Een plank fruit en brood',
-                          }
-                    },
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q2A3.png',
-                            alt: 'Pizza',
-                          }
-                    },
-                ],                    
-            },
-            3: {
-                question: 'Welke beleg jij het liefst?',
-                imgAnswers: true,
-                answers: [
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q3A1.png',
-                            alt: 'Bruinbrood',
-                          }
-                    },
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q3A2.png',
-                            alt: 'Stokbrood',
-                          }
-                    },
-                    {
-                        value: 0,
-                        img: {
-                            src: '/assets/img/questions/Q3A3.png',
-                            alt: 'Witbrood',
-                          }
-                    },
-                ],                  
-            },
-        },
+export const questions = [
+    {
+        question: 'Laat je graag eten thuis bezorgen?',
+        number: 1,
```

Comment:
A number attribute is not necessary in an array because you can use the index

Reply:
I removed the number variable and added in question the index prop for array index